client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -69,6 +71,42 @@ export default class AccountOrders extends Vue{
 
   mounted() {
     this.fetchOrders();
+    this.resizeTableColumns();
+  }
+
+  resizeTableColumns() {
+    let thElm: any;
+    let startOffset: number;
+
+    Array.prototype.forEach.call(
+      document.querySelectorAll("table th"),
+      function (th: any) {
+        th.style.position = 'relative';
+
+        const grip = document.createElement('div');
```

Comment:
May we create template for it? If not, may you use single objects like
`
grip.style = { top: 0, right: 0 ... }
...
`